platforms/db2/src/main/java/io/ebean/platform/db2/DB2RowNumberBasicLimiter.java

```diff
@@ -0,0 +1,31 @@
+package io.ebean.platform.db2;
+
+import io.ebean.config.dbplatform.BasicSqlLimiter;
+
+/**
+ * Row limiter for Oracle 9,10,11 using rownum.
+ */
+final class DB2RowNumberBasicLimiter implements BasicSqlLimiter {
+
+  @Override
+  public String limit(String dbSql, int firstRow, int maxRows) {
+    if (firstRow < 1 && maxRows < 1) {
+      return dbSql;
+    }
+    StringBuilder sb = new StringBuilder(60 + dbSql.length());
```

Comment:
```suggestion
    StringBuilder sb = new StringBuilder(120 + dbSql.length());
```
60 is not enough here. The appended constant strings has at least 89 chars